src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Startup.cs

```diff
@@ -62,14 +63,23 @@ public void ConfigureServices(IServiceCollection services)
 
         // MVC
 
+        services.AddRequestTimeouts(options =>
+        {
+            options.DefaultPolicy =
+                new RequestTimeoutPolicy
+                {
+                    Timeout = TimeSpan.FromMilliseconds(configuration.GetValue<int?>("RequestTimeoutMilliseconds") ?? 30000),
```

Comment:
Might be good for us to extract out an `AppSettingsOptions` class to house this configuration. We already do this in other parts of the service and would give some type-safety over this.

This could then be re-used in the tests as well.

Reply:
I was thinking of doing this, but then thought since it's just a one-off setting and I couldn't envisage us nesting any other options beneath the 'timeout' section as such, I thought I'd just simplify it by removing the need for the extra class. But I agree, it might be slightly better to do this. I've made a change :)